content/en/apps/guides/hosting/4.x/backups.md

```diff
@@ -0,0 +1,123 @@
+This guide is about backups in CHT 4.x - there's the [self hosted guide for 3.x]({{< relref "apps/guides/hosting/3.x/self-hosting#backup" >}}) which includes backups for 3.x.
+{{% /alert %}}
+
+## Introduction 
+
+As CHT 4.x uses a container per service, the only data that needs to be backed up is:
+
+* CouchDB database
+* Docker Compose and `.env` files 
+* TLS certificates
+
+This is because Docker containers are inherently stateless so all the important binaries are already stored in [CHT's Docker images](https://gallery.ecr.aws/s5s3h4s7/).  Docker Compose files, including the `.env` file, store all of your deployment's configuration.  Finally, the TLS certificates should be backed up to ease recover speed.
+
+How to backup each of these three pieces of data is covered below.
+
+Therefore, you do **not** need to back up the docker images for:
+
+* nginx
+* sentinel
+* api
+* haproxy
+* couchdb
+* healthcheck
+* upgrade-service
+
+## Assumptions
+
+This guide assumes you have an Ubuntu server running CHT 4.x in Docker as described in our [Self Hosting in CHT 4.x - Single CouchDB Node]({{< relref "apps/guides/hosting/4.x/self-hosting-single-node" >}}) guide. If you run `docker ps --format '{{.Names}}'` you should see something like this:
+
+```
+cht_nginx_1
+cht_sentinel_1
+cht_api_1
+cht_haproxy_1
+cht_healthcheck_1
+cht_couchdb_1
+upgrade-service-cht-upgrade-service-1
+```
+
+If you run `docker volume ls` you should see something like this:
+
+```
+DRIVER    VOLUME NAME
+local     cht_cht-credentials
+local     cht_cht-ssl
+```
+
+**Note** - In the volumes listed above, there is no volume for CouchDB data.  This is because the compose file declares this as a [bind mount](https://docs.docker.com/storage/bind-mounts/). Bind mounts use the host file system directly and do not show up in `docker volume ls` calls.  It's therefore assumed your CouchDB data location is declared in `/home/ubuntu/cht/upgrade-service/.env` which sets it with `COUCHDB_DATA=/home/ubuntu/cht/couchdb`.
+
+You should have SSH access to the server with `root` access.
+
+### Backup software
+
+It's assumed you are using which ever tool you're familiar with which might include [rsync](https://rsync.samba.org/examples.html), [borg](https://borgbackup.readthedocs.io/en/stable/) or other solution.  The locations of the backups should follow the 3-2-1 rule:
+
+> There should be at least 3 copies of the data, stored on 2 different types of storage media, and one copy should be kept offsite, in a remote location. _- [Wikipedia](https://en.wikipedia.org/wiki/Backup)_
+
+## CouchDB
+
+{{% alert title="Note" %}}
+CouchDB backups, by necessity, will have PII and PHI.  They should be safely stored to prevent unauthorized access.
+{{% /alert %}}
+
+Assuming your CouchDB is stored in `/home/ubuntu/cht/couchdb`, you should use these steps to back it up:
+
+1. Stop CouchDB.  This ensures when you back up the data files, you have a complete copy of all data in a known good state:
```

Comment:
I don't think this is necessary, and we want people to take backups regularly, so will be highly disruptive. Official documentation here: https://docs.couchdb.org/en/stable/maintenance/backups.html

Reply:
good thinking! fixed in 3f1fd4f